Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -89,6 +97,19 @@ private void HandleDeleteTrigger(EntityUid uid, DeleteOnTriggerComponent compone
             args.Handled = true;
         }
 
+        private void HandleGibTrigger(EntityUid uid, GibOnTriggerComponent component, TriggerEvent args)
+        {
+            //Code so implants can properly handle gibbing their owners
+            if (TryComp<SubdermalImplantComponent>(uid, out var implantComponent) && implantComponent.EntityUid != null)
+                _body.GibBody(implantComponent.EntityUid.Value, deleteItems: component.DeleteItems);
```

Comment:
Should this just gib the parent container? Also this should maybe be an event instead but I guess we'll see if another caller wants to also subscribe to this (in which case it should for sure be its own event).